<launch>
  <node pkg="move_arm" type="move_arm_simple_action" name="move_right_arm">    
   
    <remap from="robot_description" to="robot_description" />
    <remap from="joint_state" to="joint_states" />
    <remap from="arm_ik" to="amigo_right_arm_kinematics/get_constraint_aware_ik" />	
    <remap from="arm_fk" to="amigo_right_arm_kinematics/get_fk" />	
    <remap from="filter_trajectory" to="trajectory_filter/filter_trajectory_with_constraints" />
    <!--remap from="filter_trajectory" to="chomp_planner_longrange/filter_trajectory_with_constraints" /-->
    
    <remap from="get_trajectory_validity" to="environment_server/get_trajectory_validity" />	
    <remap from="get_environment_safety" to="environment_server/get_environment_safety" />	
    <remap from="get_execution_safety" to="environment_server/get_execution_safety" />	
    <remap from="get_group_info" to="environment_server/get_group_info" />	
    <remap from="get_robot_state" to="environment_server/get_robot_state" />	
    <remap from="get_state_validity" to="environment_server/get_state_validity" />	

    <remap from="head_monitor_action" to="head_monitor_right_arm/monitor_action" />
    <remap from="head_look_action" to="head_monitor_right_arm/look_action" />

    <param name="group" type="string" value="right_arm" />
    <param name="ik_allowed_time" type="double" value="2.0" />
    <param name="trajectory_filter_allowed_time" type="double" value="2.0" />
    <param name="group" type="string" value="right_arm" />
    <param name="controller_action_name" type="string" value="/r_joint_trajectory_action" />
    
    <param name="pause_allowed_time" type="double" value="5.0" />
    
  
    
  </node>
</launch>
